lib/widgets/atoms/profile_chip.dart

```diff
@@ -6,24 +6,28 @@ class ProfileChip extends StatelessWidget {
   final String text;
   final IconData? icon;
   final bool? usePrimaryColor;
+  final bool? applyIconColor;
 
   const ProfileChip({
     super.key,
     required this.text,
     this.icon,
     this.usePrimaryColor,
+    this.applyIconColor,
   });
 
   @override
   Widget build(BuildContext context) {
     final ThemeData theme = Theme.of(context);
+    Color iconColor = Color(theme.colorScheme.outline.value);
+    if (applyIconColor != null && applyIconColor!) {
```

Comment:
nit: You can provide a default value for optional fields in the constructor like: `this.applyIconColor = true`, that way it will never be null and you don't have to do checks like these.

Reply:
Great point! Thanks for the tip. I will add this fix in my next PR to save an approval here :)